README.md

```diff
@@ -154,19 +209,44 @@ The command line utility for Zim:
   * Want to upgrade `zimfw` to its latest version? Run `zimfw upgrade`.
   * For more information about the `zimfw` utility, run `zimfw help`.
 
-Settings
---------
+FAQ
+----
+
+### 1. How can I change default `zimrc` file location?
+
+The location of `.zimrc` file is controlled by `ZIM_HOME` environment variable. Change it to the desired path.
```

Comment:
fixed. So, the only way is to change `ZDOTDIR` ?

Reply:
Yes. The idea is that ZIM_HOME contains only downloaded stuff, and it can be recreated by downloading zimfw.sh into it. So as .zimrc is a configuration file (that you of course cannot rebuild), it does not go in there. Makes sense?